app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Successfully created acount!"
+      session[:user_id] = @user.id
+      redirect_to dashboard_path
+    end
```

Comment:
@drod1000 I think we should add an error message here flash[:danger] with the details what are missing the negative case when the visitor would be missing to fill in the registration form completely. Otherwise the SessionController looks great! FYI @Dpalazzari @njgheorghita